edx_repo_tools/codemods/node16/gha_ci_modernizer.py

```diff
@@ -0,0 +1,81 @@
+"""
+Node Modernizer for Github Actions CI
+"""
+import click
+from copy import deepcopy
+
+from edx_repo_tools.utils import YamlLoader
+
+ALLOWED_NODE_VERSIONS = [12, 14, 16]
+ALLOWED_NPM_VERSION = '8.x.x'
+
+
+class GithubCINodeModernizer(YamlLoader):
+    def __init__(self, file_path):
+        super().__init__(file_path)
+        self.yml_instance.default_flow_style = None
+        self.yml_instance.indent(mapping=2, sequence=2, offset=0)
```

Comment:
why don't we change the indentation scheme in the base class [`YamlLoader`](https://github.com/openedx/repo-tools/blob/master/edx_repo_tools/utils.py#L44) instead